.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
Good job, but you need to add a new line at the end of this file

Reply:
Hound CI is working fine, good job

Reply:
@Oloyedesinmiloluwa, I have addressed the issue you raised.